packages/grafana-prometheus/src/components/PrometheusMetricsBrowser.tsx

```diff
@@ -214,6 +217,10 @@ export class UnthemedPrometheusMetricsBrowser extends React.Component<BrowserPro
     this.setState({ metricSearchTerm: event.target.value });
   };
 
+  onChangeSeriesLimit = (event: ChangeEvent<HTMLInputElement>) => {
+    this.setState({ seriesLimit: event.target.value });
```

Comment:
It's an edge case but should we trim the value? A space might accidentally be passed and cause confusion.

Reply:
Will trim!